src/Database/Adapter/SQLite.php

```diff
@@ -53,7 +53,11 @@ public function exists(string $database, ?string $collection): bool
 
         $stmt->execute();
 
-        $document = $stmt->fetch();
+        $document = $stmt->fetchall();
```

Comment:
Let's make sure we use proper casing:

```suggestion
        $result = $stmt->fetchAll();
```